tier4/tier4_perception_dataset
# docs: comlops dataset design

## Description
This PR expands the T4 dataset format to support the Co-MLOps dataset format. 
The `docs/t4_format_3d_detailed.md` document has been updated to reflect these changes.


### Changes Made:
- Updated `docs/t4_format_3d_detailed.md` to include support for Co-MLOps dataset format.
- Provided detailed descriptions of the new dataset format fields and structures.
- UPdated README for this repositpory

## How to review
- Open the updated document: docs/t4_format_3d_detailed.md.
- Verify that the new Co-MLOps dataset format section is clear and comprehensive.
- Ensure that all new fields and structures are well-documented and aligned with the existing T4 dataset format.


## How to test
No testing is required for this documentation update.
